lib/metatags/CustomTags.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+var Backbone = require("../application/Backbone");
+
+var CustomTags = function(pairs) {
+
+    if (!pairs) {
+        throw new Error("You must provide name-content pairs when creating Metatags");
+    }
+
+    var html = "";
+    var that = this;
+
+    Object.keys(pairs).forEach(function(name) {
+        html += that.template(name, pairs[name]);
+    });
+
+    this.html = html;
+};
+
+CustomTags.prototype = {
+    template: function() {
+        throw new Error("No template implemented.");
+    },
+    html: null
+};
+
+CustomTags.extend = Backbone.Model.extend;
```

Comment:
Is Backbone important here or can we make use of underlying lodash/underscore for the extend method?